taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -99,37 +121,51 @@ const StartProject = ({ clearNavBarLinks }) => {
           </FormControl>
         </div>
 
-        <div>
-          <TextField
-            error={isInvalidBranchName}
-            helperText={
-              isInvalidBranchName &&
-              "Special characters, capital letters and white spaces are not allowed"
-            }
-            size="small"
-            sx={{ width: 265, mt: 2 }}
-            onChange={(event) => {
-              setBranchName(event.target.value);
-            }}
-            value={branchName}
-            variant="outlined"
-            label="Branch Name"
-          />
-        </div>
+        <TextField
+          error={isOwnerNameInvalid(ownerName)}
+          helperText={
+            isOwnerNameInvalid(ownerName) &&
+            "Special characters are not allowed"
+          }
+          size="small"
+          sx={{ width: 265, mt: 2 }}
+          onChange={(event) => {
+            setOwnerName(event.target.value);
+          }}
+          value={ownerName}
+          variant="outlined"
+          label="Your Name"
+          required={true}
+        />
 
-        <div>
-          <TextField
-            sx={{ width: 265, mt: 2 }}
-            minRows={4}
-            multiline
-            onChange={(event) => {
-              setDescription(event.target.value);
-            }}
-            value={description}
-            variant="outlined"
-            label="Description"
-          />
-        </div>
+        <TextField
+          error={isInvalidBranchName}
+          helperText={
+            isInvalidBranchName &&
+            "Special characters, capital letters and white spaces are not allowed"
+          }
+          size="small"
+          sx={{ width: 265, mt: 2 }}
+          onChange={(event) => {
+            setBranchName(event.target.value);
+          }}
+          value={branchName}
+          variant="outlined"
+          label="Branch Name"
+          required={true}
+        />
+
+        <TextField
```

Comment:
There again it would be great to explain what we expect.

eg:
Explain what is your goal with this new project, what changes are you going to bring. Remember to privilege small projects (do big project as a succession of small one).